pkgs/applications/misc/nut/default.nix

```diff
@@ -40,7 +47,7 @@ stdenv.mkDerivation rec {
 
   postInstall = ''
     wrapProgram $out/bin/nut-scanner --prefix LD_LIBRARY_PATH : \
-      "$out/lib:${neon}/lib:${libusb-compat-0_1.out}/lib:${avahi}/lib:${freeipmi}/lib"
```

Comment:
Can we now drop this wrapping entirely?

Reply:
Looks like it's working without the wrapper. Removed it.